packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -51,6 +52,11 @@ export interface FlexTreeContext extends ISubscribable<ForestEvents> {
 	 * The forest containing the tree data associated with this context
 	 */
 	readonly forest: IForestSubscription;
+
+	/**
+	 * The checkout object associated with this context.
+	 */
+	readonly checkout: ITreeCheckout;
```

Comment:
How well is `ITreeCheckout` documented? Just wondering if the docs here sufficiently describe how users would consume this property.